src/app/games/siam/tests/siam.component.spec.ts

```diff
@@ -272,4 +271,55 @@ describe('SiamComponent', () => {
         const move: SiamMove = SiamMove.from(5, 4, MGPOptional.of(Orthogonal.LEFT), Orthogonal.LEFT).get();
         await testUtils.expectMoveSuccess('#indicator_4_4_LEFT', move);
     }));
+
+    function expectTranslationYToBe(transform: SVGTransform, y: number): void {
+        expect(transform.type).toBe(SVGTransform.SVG_TRANSFORM_TRANSLATE);
+        // In a SVG transform, f is the y coordinate
+        expect(transform.matrix.f).toBe(y);
+    }
+    it('should display current player pieces on the bottom (Player.ONE)', fakeAsync(async() => {
+        // Given a state from the point of view of player 1
+        const board: Table<SiamPiece> = [
+            [_, _, _, _, _],
+            [_, _, _, _, _],
+            [_, M, M, M, _],
+            [_, _, _, _, _],
+            [_, _, _, _, U],
+        ];
+        const state: SiamState = new SiamState(board, 1);
```

Comment:
so, this is not in the point of view of player one here !
Et je suggère d'utilise Player.ONE même dans les commentaires :p ! Ou autres usages déjà présent, mais pas player 1 :D !

Reply:
commentaire doublon, cf. message précédent pour la réponse/question. Et je fixe le player 1

Reply:
donc voilà, à checker après avoir fix setupSteak

Reply:
fait !